kernel/aster-nix/src/net/socket/ip/datagram/mod.rs

```diff
@@ -280,12 +280,14 @@ impl Socket for DatagramSocket {
     fn recvfrom(&self, buf: &mut [u8], flags: SendRecvFlags) -> Result<(usize, SocketAddr)> {
         debug_assert!(flags.is_all_supported());
 
-        poll_ifaces();
-        if self.is_nonblocking() {
+        let recv_bytes = if self.is_nonblocking() {
```

Comment:
The returned type of `try_recvfrom` is `(usize, addr)`. So maybe renamed the variable `recv_bytes` as `(recv_bytes, peer_addr)` or `received`(just like `accepted`) is better.